build/docker/intel-qat-plugin-kerneldrv.Dockerfile

```diff
@@ -20,6 +20,18 @@
 # The main branch defaults to using the latest Golang base image.
 ARG GOLANG_BASE=golang:1.17-bullseye
 
+# FINAL_BASE can be used to configure the base image of the final image.
+#
+# This is used in two ways:
+# 1) make <image-name> BUILDER=<docker|buildah>
+# 2) docker build ... -f <image-name>.Dockerfile
+#
+# The project default is 1) which sets FINAL_BASE=gcr.io/distroless/static
+# (see build-image.sh).
+# Declaring FINAL_BASE ARG but not setting the value to resolve build warning:
+# "[Warning] one or more build args were not consumed: [FINAL_BASE]"
+ARG FINAL_BASE
```

Comment:
The same here. Am I missing something here?

Reply:
did you read lines 31:32?

Reply:
missed it. ok, makes sense now, thanks.